Formula/gmp.rb

```diff
@@ -16,12 +16,13 @@ class Gmp < Formula
 
   def install
     ENV.cxx11 if build.cxx11?
-    args = ["--prefix=#{prefix}", "--enable-cxx"]
 
-    # https://github.com/Homebrew/homebrew/issues/20693
-    args << "--disable-assembly" if build.bottle?
+    # XNU release major version
+    osmajor = MacOS.version.to_s[3..-1].to_i + 4
```

Comment:
That's the full version. `uname -r`.chomp
=> "12.6.0"

So you still end up having to do string manipulations then to extract the major version. Running processes for answers we already know is pretty gross lol